### ISSUE TITLE
error calculation is off

### ISSUE THREAD

**Issue by pbielefeldt** [pbielefeldt/strip_response_resolution#3]
Not sure why, but now I get a seg violation.

```bash
 *** Break *** segmentation violation
 Generating stack trace...
 0x00007f98eaff4490 in G__exec_asm + 0x43a0 from /usr/lib64/root/libCint.so.5.34
 0x00007f98eaff793d in G__exec_bytecode + 0x69d from /usr/lib64/root/libCint.so.5.34
 0x00007f98eaf94aea in G__interpret_func + 0x23da from /usr/lib64/root/libCint.so.5.34
 0x00007f98eafaac1c in G__getfunction + 0x1c6c from /usr/lib64/root/libCint.so.5.34
 0x00007f98eafb3596 in G__getitem + 0x6f6 from /usr/lib64/root/libCint.so.5.34
 0x00007f98eafb7ad0 in G__getexpr + 0x3e20 from /usr/lib64/root/libCint.so.5.34
 0x00007f98eafd0416 in G__define_var + 0x2f66 from /usr/lib64/root/libCint.so.5.34
 0x00007f98eaf4597a in G__exec_statement + 0x36ea from /usr/lib64/root/libCint.so.5.34
 0x00007f98eaf4de47 in <unknown> from /usr/lib64/root/libCint.so.5.34
 0x00007f98eaf4c5b4 in G__exec_statement + 0xa324 from /usr/lib64/root/libCint.so.5.34
 0x00007f98eaf95386 in G__interpret_func + 0x2c76 from /usr/lib64/root/libCint.so.5.34
 0x00007f98eafaac1c in G__getfunction + 0x1c6c from /usr/lib64/root/libCint.so.5.34
 0x00007f98eafb3596 in G__getitem + 0x6f6 from /usr/lib64/root/libCint.so.5.34
 0x00007f98eafb7ad0 in G__getexpr + 0x3e20 from /usr/lib64/root/libCint.so.5.34
 0x00007f98eafc287c in G__calc_internal + 0x42c from /usr/lib64/root/libCint.so.5.34
 0x00007f98eaf3b66f in G__process_cmd + 0x2c2f from /usr/lib64/root/libCint.so.5.34
 0x00007f98ec5dcb66 in TCint::ProcessLine(char const*, TInterpreter::EErrorCode*) + 0x3f6 from /usr/lib64/root/libCore.so.5.34
 0x00007f98ec5d7d97 in TCint::ProcessLineSynch(char const*, TInterpreter::EErrorCode*) + 0x87 from /usr/lib64/root/libCore.so.5.34
 0x00007f98ec5828bc in TApplication::ExecuteFile(char const*, int*, bool) + 0x4fc from /usr/lib64/root/libCore.so.5.34
 0x00007f98ec583c62 in TApplication::ProcessLine(char const*, bool, int*) + 0x8c2 from /usr/lib64/root/libCore.so.5.34
 0x00007f98ec159af8 in TRint::Run(bool) + 0x248 from /usr/lib64/root/libRint.so.5.34
 0x00005624a909c34c in main + 0x4c from /usr/bin/root.exe
 0x00007f98eb7fcf4a in __libc_start_main + 0xea from /lib64/libc.so.6
 0x00005624a909c3aa in _start + 0x2a from /usr/bin/root.exe
Root > Function num_strips() busy flag cleared
Function StripResolution() busy flag cleared
```

**Comment by pbielefeldt** [comment]
Of course.
`plane` is deleted within every event. 
The way we calculated any error does not work for multiple events per run (cf. #1).
Maybe `RMS2` and `sigma` in 6d5eaa554fa3e7dc08dd63689c3452baf672662b are the better measurement anyway?

**Comment by pbielefeldt** [comment]
Solved in 5011770a9f7ec812c72b6dca99f4acdf0d8333b1.
Yet no good measure for error though.
